### DISCUSSION TITLE
Local Wordpress

### DISCUSSION THREAD

**Discussion by ghost** [thebaselab/codeapp#984] [Q&A]
Hi there, just started using code app and it's definitely been great so far. I was curious as I'm a newbie to a lot of this, is it possible to make a local Wordpress dev environment on my iPad using this app? If so, could someone point me in the right direction to accomplish it?

**Comment by ghost** [comment]
Just a quick update. I created a Wordpress install on my computer, used the SQLite integration plugin to create the SQLite database. I then copied the install to my iPad and started the php server. 

It is starting to work, the front end loads fine, however, after I sign in to access the backend, I get an error of a call to undefined function, gzinflate. From what I can tell, this is because php in Code App does not have the Zlib extension and I can't seem to install it. 

So maybe any thoughts, or how can I install the zlib extension?

**Reply by bummoblizard** [reply]
Thanks for sharing! I will need to re-compile php to include the extension. I'll let you know the result.

**Comment by bummoblizard** [comment]
PHP 8.3.2 with Zlib support will be available on TestFlight shortly.

**Reply by ghost** [reply]
That's great. I bought through the App Store and cannot join via test flight as it says it is full. If you have another way so I can test, can you let me know. Thank you.

**Reply by bummoblizard** [reply]
@SRUKen I have opened up more slots.

**Reply by ghost** [reply]
I'm still playing around with this but even adding the module definitely increases what I can do. I noticed a few things, there's some other modules Wordpress needs according to site health such as: curl, exif, imagick, mbstring, openssl, zip, is not installed, gd, iconv, and intl. Also, if it is possible to adjust php options, like maximum file upload sizes or maximum execution time could make it more usable. Let me know if I'm missing anything to do these myself in the app. Thank so much for the work you've done on this.
